profiles/killbill/pom.xml

```diff
@@ -66,15 +66,10 @@
             <artifactId>jsr305</artifactId>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <scope>compile</scope>
-        </dependency>
         <dependency>
             <groupId>com.google.inject</groupId>
             <artifactId>guice</artifactId>
-            <scope>compile</scope>
+            <scope>provided</scope>
```

Comment:
Interesting...

Reply:
No, this is not true. Problem is, some dependencies still require guava (`testing-postgresql-server`, `embedded-redis` etc, including `killbill-platform` modules. So maybe `killbill-profiles-killbill` is the last modules that will have guava dependency, waiting fix in `killbill-platform` and `killbill-client-java`.

Reply:
Now that you have completed `killbill-commons`, I think we should then (try to) eliminate guava from `killbill-platform`, and then complete the work on the `killbill` repo. @xsalefter Does this make sense?